umap/static/umap/js/umap.js

```diff
@@ -1769,9 +1769,24 @@ U.Map = L.Map.extend({
         callback: this.openExternalRouting,
       })
     }
+    if (this.options.urls.edit_in_osm) {
+      items.push('-', {
+        text: L._('Edit in OpenStreetMap'),
+        callback: this.editInOSM,
+      })
+    }
     this.options.contextmenuItems = items
   },
 
+  editInOSM: function (e) {
+    const url = this.urls.get('edit_in_osm', {
+      lat: e.latlng.lat,
+      lng: e.latlng.lng,
+      zoom: Math.max(this.getZoom(), 16),
+    })
+    if (url) window.open(url)
```

Comment:
Not specific to that PR but I think we should open a new tab/window in these cases (probably applies to `openExternalRouting` too):

```suggestion
    if (url) window.open(url, '_blank')
```

Reply:
That's a good point, but strangely it already opens in a new window (same for routing).